blog/2016-10-06-fusetooling-release-mars.adoc

```diff
@@ -0,0 +1,45 @@
+= JBoss Fuse Tooling released for Eclipse Mars 
+:page-layout: blog
+:page-author: lhein
+:page-tags: [fuse, camel, integration, jbosstools, devstudio, jbosscentral]
+
+
```

Comment:
somewhere here, we need to put a link to the same article which is planned to be published on developers.red.com